MEMBER_EXPECTATIONS.md

```diff
@@ -7,7 +7,7 @@ It is understood that members will have individual opinions and will express
 their individual opinions during discussions within the organization. All
 contributors are expected to be respectful of these different viewpoints,
 and to work collaboratively with one another in a manner that constructively
-elevates all contributors.
+elevates all contributors. If a member has an objection or dissent they should express it as early as possible to ensure there is ample time to discuss and reach consensus.
```

Comment:
I would prefer to soften the language here, from “should” to “it is preferred” or “it is recommended”. 

I think we should allow objections to be brought up as late as possible, as people may travel/be off/whatever. Keeping it vague (asap) would allow some possible misunderstanding. Somebody might wonder “am I too late to object?”.

Reply:
If they’re traveling, then it’s as soon as possible whenever they’re back - this already seems very soft to me.